x-pack/plugins/screenshotting/server/layouts/create_layout.test.ts

```diff
@@ -61,6 +61,7 @@ describe('Create Layout', () => {
         },
         "useReportingBranding": true,
         "viewport": Object {
+          "deviceScaleFactor": 1,
```

Comment:
I hardened the types in this PR, to make sure the scale factor is always part of the viewport info